srtcore/packetfilter_api.h

```diff
@@ -0,0 +1,135 @@
+/*
+ * SRT - Secure, Reliable, Transport
+ * Copyright (c) 2018 Haivision Systems Inc.
+ * 
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * 
+ */
+
+#ifndef INC__PACKETFILTER_API_H
+#define INC__PACKETFILTER_API_H
+
+enum SrtPktHeaderFields
+{
+    SRT_PH_SEQNO = 0, //< sequence number
+    SRT_PH_MSGNO = 1, //< message number
+    SRT_PH_TIMESTAMP = 2, //< time stamp
+    SRT_PH_ID = 3,  //< socket ID
+
+    // Must be the last value - this is size of all, not a field id
+    SRT_PH__SIZE
+};
+
+
+enum SRT_ARQLevel
+{
+    SRT_ARQ_NEVER,   //< Never send LOSSREPORT
+    SRT_ARQ_ONREQ,  //< Only record the loss, but report only those that are returned in receive()
+    SRT_ARQ_ALWAYS, //< always send LOSSREPORT immediately after detecting a loss
```

Comment:
Align block of comments (first comment is one extra space to the right)

Reply:
Should be outdated now, too.